run_uh.py

```diff
@@ -50,19 +50,15 @@
 # this will break any run_uh imports from other locations (e.g. _get_version())
 
 def exit_with_error(title, message):
-	if title + '.' != message:
-		print(title)
-	print(message)
-
+	print(title + '\n' + message);
 	try:
 		import Tkinter
 		import tkMessageBox
 		window = Tkinter.Tk()
 		window.wm_withdraw()
 		tkMessageBox.showerror(title, message)
 	except ImportError:
-		# tkinter may be missing
-		pass
+		print("Tkinter module missing. Install Tkinter for Python2.7 to view pop-up error dialogs");
```

Comment:
I'd like to see this visually separated from the actual error message. Since the module is optional, it shouldn't shadow the actual problem. Right now, it'll probably look like this:

```
Unsupported Python version
32 bit FIFE requires 32 bit (x86) Python 2.
Tkinter module missing. Install Tkinter for Python2.7 to view pop-up error dialogs
```

Looking at this, title and message should stand out more. Perhaps like this:

```
Error: Unsupported Python version
32 bit FIFE requires 32 bit (x86) Python 2.
Tkinter module missing. Install Tkinter for Python2.7 to view pop-up error dialogs
```

I suppose it's enough to add a new empty line:

```
Error: Unsupported Python version
32 bit FIFE requires 32 bit (x86) Python 2.

Tkinter module missing. Install Tkinter for Python2.7 to view pop-up error dialogs
```

I was going to suggest rewording the sentence, but then again.. this function is used a total of three times. Two times to show an error about python and one time about fife. Is it even *necessary* to ask the user to install tkinter?